lib/app/workorder.js

```diff
@@ -34,6 +36,20 @@ module.exports = function(mediator) {
     workorder.startTimestamp = newDate.getTime();
     workorder.finishTimestamp = tomorrow.getTime();
   });
-
-  return store.init('workorders', workorders, mediator);
+  
+  return store.init('workorders', workorders, mediator, function(dataStore, dataTopicPrefix) {
+    return dataStore.listen(dataTopicPrefix, mediator, {
+      // Override list method for workorders
+      list: function(filter) {
+        if (filter.sessionToken) {
+          return userSessionApi.getUserIdForSessionPromise(filter.sessionToken).then(function(userId) {
+            // Override filter
+            filter.value = userId;
+            return dataStore.list(filter);
```

Comment:
Shouldn't this be `this.model.list(filter)` ?

From https://github.com/feedhenry-raincatcher/raincatcher-mongoose-store#listen